src/components/FileList/FileList.js

```diff
@@ -20,17 +31,164 @@ const FileList = ({fileList, studyId}) => {
+    var sortedList = [];
+    if (
+      sortMethod === '' &&
+      typeFilterStatus === '' &&
+      approvalFilterStatus === ''
+    ) {
+      sortedList = fileList
+        .filter(obj => fileLatestStatus(obj.node) === 'CHN')
+        .concat(fileList.filter(obj => fileLatestStatus(obj.node) !== 'CHN'));
+    } else {
+      if (sortMethod === 'createDate') {
+        sortedList = fileList.sort((a, b) =>
+          Date.parse(fileOldestDate(fileSortedVersions(a.node))) >
+          Date.parse(fileOldestDate(fileSortedVersions(b.node)))
+            ? 1
+            : Date.parse(fileOldestDate(fileSortedVersions(b.node))) >
+              Date.parse(fileOldestDate(fileSortedVersions(a.node)))
+            ? -1
+            : 0,
+        );
+      } else {
+        sortedList = fileList.sort((a, b) =>
+          Date.parse(fileLatestDate(fileSortedVersions(a.node))) >
+          Date.parse(fileLatestDate(fileSortedVersions(b.node)))
+            ? 1
+            : Date.parse(fileLatestDate(fileSortedVersions(b.node))) >
+              Date.parse(fileLatestDate(fileSortedVersions(a.node)))
+            ? -1
+            : 0,
+        );
+      }
+      if (sortDirection === 'descending') {
+        sortedList = sortedList.reverse();
+      }
+      if (typeFilterStatus !== '') {
+        sortedList = sortedList.filter(
+          obj => obj.node.fileType === typeFilterStatus,
+        );
+      }
+      if (approvalFilterStatus !== '') {
+        sortedList = sortedList.filter(
+          obj => fileLatestStatus(obj.node) === approvalFilterStatus,
+        );
+      }
+    }
+    return sortedList;
+  };
+
+  const pageCount = Math.ceil(sortedFileList().length / perPage);
+  const pageItems = sortedFileList().slice(
     perPage * (page - 1),
     perPage * (page - 1) + perPage,
   );
 
   return (
     <Fragment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="right"
+      >
+        <Dropdown
+          selection
+          clearable
+          value={typeFilterStatus}
+          options={typeOptions}
+          placeholder="Choose a file type"
+          onChange={(e, {value}) => {
+            setTypeFilterStatus(value);
+          }}
+        />
+      </Segment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="right"
+      >
+        <span className="smallLabel">Filter by:</span>
+        <Dropdown
+          selection
+          clearable
+          value={approvalFilterStatus}
+          options={statusOptions}
+          placeholder="Choose an approval status"
+          onChange={(e, {value}) => {
+            setApprovalFilterStatus(value);
+          }}
+        />
+      </Segment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="right"
+      >
+        <span className="smallLabel">Sorted by:</span>
+        <Dropdown
```

Comment:
Clicking the dropdowns will automatically select the first element when clicking out of the select box. This is a bit obnoxious as I don't usually expect to have a filter applied if I 'cancel' the select. I think this can be disabled through setting `selectOnBlur` to `false`.